staging/vhost-device-gpu/rutabaga_gfx/src/rutabaga_core.rs

```diff
@@ -50,6 +50,12 @@ pub struct RutabagaResource {
     pub size: u64,
     pub mapping: Option<MemoryMapping>,
 }
+// SAFETY: Safe as the structure can be sent to another thread.
+unsafe impl Send for RutabagaResource {}
+
+// SAFETY: Safe as the structure can be shared with another thread as the state
+// is protected with a lock.
```

Comment:
Which lock?

Reply:
That was an  oversight, I need to change that comment. thanks for catching it.